# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1429549274 -7200
#      Mon Apr 20 19:01:14 2015 +0200
# Node ID 6288272b89d3a3064a0268211cce1e394155e37a
# Parent  5e0051460718dba2619b8c8d18a4fa1a592a527d
de_DE.ts: Update german translation

diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -3158,37 +3158,37 @@ a:link { text-decoration: underline; col
     <message>
         <location line="-22"/>
         <source>Goto Line</source>
         <translation>Gehe zu Zeile</translation>
     </message>
     <message>
         <location line="-125"/>
         <source>Step</source>
-        <translation type="unfinished">Einzelschritt</translation>
+        <translation>Einzelschritt</translation>
     </message>
     <message>
         <location line="+36"/>
         <source>Reset Default Window Layout</source>
-        <translation type="unfinished">Fensterlayout auf Grundeinstellung zurücksetzen</translation>
+        <translation>Fensterlayout auf Grundeinstellung zurücksetzen</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Report Bug</source>
-        <translation type="unfinished">Fehler melden</translation>
+        <translation>Fehler melden</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>Close Other Files</source>
-        <translation type="unfinished">Andere Dokumente schließen</translation>
+        <translation>Andere Dokumente schließen</translation>
     </message>
     <message>
         <location line="+30"/>
         <source>Show Completion List</source>
-        <translation type="unfinished"></translation>
+        <translation>Vervollständigungsliste anzeigen</translation>
     </message>
     <message>
         <location line="+33"/>
         <source>Move to Matching Brace</source>
         <translation>Zur schließenden Klammer springen</translation>
     </message>
     <message>
         <location line="+2"/>
